spring-boot/admin/src/main/resources/application.yml

```diff
@@ -5,78 +5,83 @@ mvn.pom.project.version: @@project.version@@
+      fix: 1
+      bookKeeping.contact: fperezduran@bebr.nl
+    call-request-fee: 0
 
-      gettingStarted:
-         consumer:
-           testService:
-# the following two elements are required to load the test service for getting started wizard
-                provider.username: BrunoSilvaATB
-                uriIdentifier: getting-started-service-consumer
-   proxy:
-      executable-jar:
-         generated.path: /tmp
-         # URL of the "template jar" that CoatRack uses as basis to generate proxies (without the trailing version number and .jar filename extension)
-         template.url: https://www.kyoe.es/proxies/coatrack-proxy-
-         path: path
-         filename-custom:
-            prefix: coatrack-gateway-
-            suffix: .jar
-      generate-bootstrap-properties:
-         spring.application.name.prefix: ygg-proxy-
-         spring.cloud.config.uri: http://localhost
-      server.port.defaultValue: 8088
+    gitService:
+      # URL and credentials for the Git repository where CoatRack stores gateway config files
+      url: http://localhost
+      user: user
+      password: password
+
+    # URL of the CoatRack web application's API that will be called by CoatRack gateways
+    api-base-url-for-gateway: http://localhost:8080/api/
+
+    gettingStarted:
+      consumer:
+        testService:
+          # Provider and identifier to identify the test service to be used in the getting started tutorial
+          provider.username: BrunoSilvaATB
+          uriIdentifier: getting-started-service-consumer
+  proxy:
+    executable-jar:
+      generated.path: /tmp
+      # URL of the "template jar" that CoatRack uses as basis to generate proxies (without the trailing version number and .jar filename extension)
+      template.url: https://www.kyoe.es/proxies/coatrack-proxy-
+      path: path
+      filename-custom:
+        prefix: coatrack-gateway-
+        suffix: .jar
+    generate-bootstrap-properties:
+      spring.application.name.prefix: ygg-proxy-
+      spring.cloud.config.uri: http://localhost:8998
+    server.port.defaultValue: 8088
 
 # Necessary to define the threshold times to display different
 # status (warning and critical) on the Gateway Health Monitor
 ygg.gateway-health-monitor:
-   warning.threshold.minutes: 5
-   critical.threshold.minutes: 60
-   
+  warning.threshold.minutes: 5
+  critical.threshold.minutes: 60
+
 spring:
   data:
     rest:
-      base-path: /api   
-      
+      base-path: /api
+
 spring.application.name: ygg-admin
 
-# Still require for the service generated proxies, maybe we can consider the idea to rename the property in order to not confuse
+# TODO Still require for the service generated proxies, maybe we can consider the idea to rename the property in order to not confuse
 spring.cloud.config:
-        username: admin
-        password: password
-        uri: https://config.coatrack.eu
+  username: admin
+  password: password
+  uri: https://config.coatrack.eu
```

Comment:
I think we should remove this credentials becuase it is pointing out to the production config server